### PR TITLE
[Doc] Add 3.2.0 RC01 release notes in CN

### PR DESCRIPTION
**Pull request by EsoragotoSpirit**
Why I'm doing:
Add 3.2.0 RC01 release notes in CN
What I'm doing:

Fixes #issue

## What type of PR is this:

- [ ] BugFix
- [ ] Feature
- [ ] Enhancement
- [ ] Refactor
- [ ] UT
- [x] Doc
- [ ] Tool

Does this PR entail a change in behavior?

- [ ] Yes, this PR will result in a change in behavior.
- [x] No, this PR will not result in a change in behavior.

If yes, please specify the type of change:

- [ ] Interface/UI changes: syntax, type conversion, expression evaluation, display information
- [ ] Parameter changes: default values, similar parameters but with different default values
- [ ] Policy changes: use new policy to replace old one, functionality automatically enabled
- [ ] Feature removed
- [ ] Miscellaneous: upgrade & downgrade compatibility, etc.

## Checklist:

- [ ] I have added test cases for my bug fix or my new feature
- [ ] This pr needs user documentation (for new or modified features or behaviors)
  - [ ] I have added documentation for my new feature or new function

## Bugfix cherry-pick branch check:

- [ ] I have checked the version labels which the pr will be auto-backported to the target branch
  - [ ] 3.2
  - [ ] 3.1
  - [ ] 3.0
  - [ ] 2.5

### CODE REVIEW

File: docs/zh/release_notes/release-3.2.md [hunk 1]
```diff
@@ -0,0 +1,118 @@
...
+- 日期函数：dayofweek_iso、week_iso、quarters_add、quarters_sub、milliseconds_add、milliseconds_sub、date_diff、jodatime_format、str_to_jodatime、to_iso8601、to_tera_date、to_tera_timestamp
+- 模糊/正则匹配函数：regexp_extract_all
+- hash 函数：xx_hash3_64
+- 聚合函数：approx_top_k
+- 窗口函数：cume_dist、percent_rank、session_number
+- 工具函数：dict_mapping、get_query_profile
+
+#### 权限
+
+支持通过 Apache Ranger 实现访问控制，提供更高层次的数据安全保障，并且允许复用原有的外部数据源 Service。StarRocks 集成 Apache Ranger 后可以实现以下权限控制方式：
```
Comment (evelynzhaojie) [inline docs/zh/release_notes/release-3.2.md:42]: 并且允许复用原有的外部数据源 Service --- 并且允许复用外部数据源对应的 Ranger Service

File: docs/zh/release_notes/release-3.2.md [hunk 2]
```diff
@@ -0,0 +1,118 @@
...
+
+- 访问 StarRocks 内表、外表或其他对象时，可根据在 Ranger 中创建的 StarRocks Service 配置的访问策略来进行访问控制。
+- 访问 External Catalog 时，也可以复用对应数据源原有的 Ranger service（如 Hive Service）来进行访问控制（当前暂未支持导出数据到 Hive 操作的权限控制）。
```
Comment (evelynzhaojie) [inline docs/zh/release_notes/release-3.2.md:45]: 当前暂未支持导出数据到 Hive 操作的权限控制

Comment (evelynzhaojie) [inline docs/zh/release_notes/release-3.2.md:45]: 当前暂未提供权限控制策略来导出数据到 Hive。

**Comment by EsoragotoSpirit** [comment]
@mergify rebase

**Review by amber-create — APPROVED** [review]

**Review by evelynzhaojie — APPROVED** [review]

**Comment by EsoragotoSpirit** [comment]
@mergify backport branch-3.2

**Pull request merged**